### ISSUE TITLE
Enhance script debugger help/quit

### ISSUE THREAD

**Issue by dnsmichi** [Icinga/icinga2#5595]
## Expected Behavior

- Highlight `$help` on first run.
- `$help` prints `$continue`, `$quit` and other variants
- `$quit` must be implemented too

Think about possible global commands which could be needed too.

## Current Behavior

Really exiting the script debugger (or continuing with config validation) is hard if you don't know exit() and Ctrl+C.

```
mbmif /usr/local/icinga2 (master *) # icinga2 console -X
Icinga 2 (version: v2.7.0-172-gfafc15a61)
<1> => debugger
Breakpoint encountered.
Location: in <1>: 1:0-1:7You can inspect expressions (such as variables) by entering them at the prompt.
To leave the debugger and continue the program use "$continue".
<1> => $quit
Unknown debugger command: $quit
<2> => $exit
Unknown debugger command: $exit
<3> => exit(1)
```

**Comment by Crunsher** [comment]
I looked at the issue and found no perfect solution...
First, the script debugger and console are the same. But respond differently from breaking from the REPL (as `$continue` does), when invoked via `icinga2 daemon -X` and hitting a breakpoint, it continues to run Icinga 2, when `$continue` is used in `icinga2 console` it quits the console.
We can't have the script debugger quit icinga2 after hitting a breakpoint, it's messy - let the user do that. So I'd suggest we leave continue unchanged but add quit and exit doing the same.

**Comment by dnsmichi** [comment]
Solved in #5979
